pulp-worker/entrypoint.sh

```diff
@@ -18,6 +19,7 @@ Name-Real: Tester
 Name-Email: test@test.com
 Expire-Date: 0
 EOF
+fi
```

Comment:
This outdented `if` bugs me more than it should. :) 

Then again, so does the heredoc. One way I might work around that is with printf. As in:

```bash
printf "%s\n" \
"Key: val" \
"other: thing" \
| gpg ...
```
Then just the extra quotes would bug me. ;)